infra/terraform/test-org/github/labels.tf

```diff
@@ -26,96 +26,21 @@ locals {
       color : "d73a4a"
       description : "Something isn't working"
     },
-    {
-      name : "duplicate",
-      color : "cfd3d7"
-      description : "This issue or pull request already exists"
-    },
-    {
-      name : "good first issue",
```

Comment:
This might be good to keep to provide new contributors with easy tasks.

Reply:
SG - added back